src/tasks/receipts.rs

```diff
@@ -0,0 +1,74 @@
+use crate::config::Provider;
+use alloy::{primitives::TxHash, providers::Provider as _};
+use metrics::{counter, histogram};
+use std::time::Instant;
+use tokio::{sync::mpsc, task::JoinHandle};
+use tracing::{debug, error};
+
+/// Submits sidecars in ethereum txns to mainnet ethereum
```

Comment:
I think this comment is a mistaken holdover from `SubmitTask` struct, should be updated to describe `ReceiptTask`.

Reply:
good catch https://github.com/init4tech/builder/pull/48/commits/7ebd8d219556a9fa66d2c3c697b445c254d6029c